LayoutTests/fast/xpath/null-namespace-in-html-expected.txt

```diff
@@ -1,7 +1,6 @@
-This tests that XPath expressions with prefixes work correctly.
-
-PASS //div
+FAIL //div incorrect length (expected 1, actual 0)
 PASS successfullyParsed is true
+Some tests failed.
 
 TEST COMPLETE
-
+This tests that XPath expressions with prefixes work correctly.
```

Comment:
It's also not great that this moved to the bottom. You may want to use the `description()` function across the board for the harness to handle the title.